README.md

```diff
@@ -27,3 +31,41 @@ The plugin for pretty printing [Qt][qt] types by [CLion][clion] debugger.
 ## Architectures tested on
 * x64
 
+## Requirements
+
+This plugin needs Debug information for Qt.
+
+If you installed Qt with Qt Online installer, ensure you have installed
+the `Qt Debug Information files`:
+
+![Checkbox for QtDebug Information files](images/Qt_Debug_Information_files_checkbox.png)
+
+If you are using Arch Linux, you need to install `qt6-base-debug` package.
+You can install manually (by specifying the url), or by enabling global repo. See [wiki](https://wiki.archlinux.org/title/Debugging/Getting_traces#Installing_debug_packages).
+```
+sudo pacman -U https://geo.mirror.pkgbuild.com/extra-debug/os/x86_64/qt6-base-debug-6.7.2-1-x86_64.pkg.tar.zst
+```
+
+## Troubleshooting
+
+### Qt Types are not pretty printed
+
+First, ensure you have satisfied [requirements](#Requirements).
+
+Now use gdb console to make a quick check if a specific version includes type metainfo by setting a breakpoint and running the command:
+
+```
+python print(gdb.parse_and_eval('*(&qtHookData)'))
```

Comment:
There is an LLDB version of the command:

`script print(lldb.target.FindFirstGlobalVariable('qtHookData').GetPointeeData(2, 1))`

Should print something like:
![image](https://github.com/user-attachments/assets/a7b37f52-dbc5-463d-806f-151e23845d04)

Reply:
Interesting, your's "(lldb)" text is green, but mine is white. I use CL-242.20224.66